packages/instant/src/components/timed_progress_bar.tsx

```diff
@@ -0,0 +1,102 @@
+import * as _ from 'lodash';
+import * as React from 'react';
+
+import { PROGRESS_FINISH_ANIMATION_TIME_MS, PROGRESS_STALL_AT_PERCENTAGE } from '../constants';
```

Comment:
Can these just be props with default values?

Reply:
I'm a little mixed on this -- this is used only in one place, and we don't expect anyone to ever use values other than these defaults, so I'm wondering if making these props is a bit of over-engineering?

Reply:
Don't feel strongly about this.